addon/webextension/selector/ui.js

```diff
@@ -95,16 +95,22 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
     return window.downloadOnly;
   }
 
-  function renderDownloadNotice() {
-    let notice = makeEl("div", "download-only");
+  function renderDownloadNotice(initAtBottom = false) {
```

Comment:
fiddled with dom structure here. added an optional param b/c the two iframes we inject this to are really different.